docs/rst/notes/notes.rst

```diff
@@ -1,53 +1,37 @@
 .. _release_notes:
 
-#############
-Version 0.1.0
-#############
-
-This is the first release of *eProsima Fast DDS Monitor*.
-
-This release includes several **features** regarding the monitoring of DDS network and the visualization
-of a real time working network, as instrumentation data related with this network:
-
-This release includes the following **visual features**
-
-* Visual Interface to monitor a DDS network.
-
-* Visualize *alive* entities in a DDS network in real time.
-
-* Represent instrumentation data of several kinds of a DDS network:
-
-  * Latency
-  * Throughput
-  * Data sent
-  * Metadata sent
-  * Discovery time
-
-* Represent instrumentation data regarding different cumulative functions:
-
-  * Mean
-  * Median
-  * Standard deviation
-  * Maximum
-  * Minimum
-  * Count
-  * Sum
-
-* Real time events:
-
-  * New entities discovered.
-  * New issues detected.
-
-This release includes the following **interactive features**
-
-* Initiate monitoring a DDS network:
-
-  * Monitoring a DDS domain.
-  * Monitoring a Discovery Server network.
-
-* Represent any data kind with any cumulative function regarding any one or two entities in the network.
-* Select an Entity in order to retrieve information about the entity configuration and its data summary.
-* User friendly modifiable layout:
-
-  * Resize windows and panels.
-  * Create, delete and move charts and panels.
+##############
+Version v0.2.0
+##############
+
+This release includes the following **improvements**:
+
+* Add ToolTips for dialog parameters
+* Add Real-Time charts
+* Refactor sidebars
+  * New left sidebar with all the DDS, physical and logical entities together with its corresponding information.
+  * Separated pane for Fast DDS Monitor status.
+  * New pane to display Issues information.
+* Possibility to change the Entity name setting an alias for it.
+* Entities not alive are marked in gray color.
+* Possibility to display/hide the not alive entities.
+* Entity menu displayed right-clicking on them.
+* Support for Windows platforms.
+
```

Comment:
I would add a comment about new update method, even when is not so visible for the user (it is a bit in the way that dds panel does not dissapear with each new entity)